apps/src/blocklyAddons/cdoWorkspaceSvg.js

```diff
@@ -33,5 +33,6 @@ WorkspaceSvg.prototype.blockSpaceEditor = {
   blockLimits: {
     blockLimitExceeded: () => false, // TODO
     getLimit: () => {} // TODO
-  }
+  },
+  svgResize: () => {} // TODO
```

Comment:
Out of curiosity, what's the implication of this not yet being TODONE?

Reply:
I'm not entirely sure yet... It seems like it's used when we embed blocks in instructions/feedback/hints to shrink the svg down to size. What's not clear to me is why using `getBBox()` and setting the width/height immediately when we create the svg wasn't sufficient. The original function was defined [here](https://github.com/code-dot-org/blockly/blob/main/core/ui/block_space/block_space_editor.js#L736)